const TaskError = () => {
  return <p>Desesperate, hubo un error...</p>;
};

export default TaskError;
